src/main/java/com/vinylteam/vinyl/service/JwtTokenProvider.java

```diff
@@ -16,53 +23,74 @@
 import javax.annotation.PostConstruct;
 import javax.crypto.SecretKey;
 import javax.servlet.http.HttpServletRequest;
-import java.util.Collection;
-import java.util.Date;
-import java.util.Objects;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.*;
 
 import static com.vinylteam.vinyl.security.SecurityConstants.AUTHORIZATION_HEADER_NAME;
 import static com.vinylteam.vinyl.security.SecurityConstants.TOKEN_PREFIX;
 
 @Service
+@RequiredArgsConstructor
 @Slf4j
 public class JwtTokenProvider implements JwtService {
 
+    private AuthenticationManager authenticationManager;
+
+    private final UserService userService;
+    private final UserMapper userMapper;
     private SecretKey secretKey;
 
-    @Value("${jwt.token.expired:120000}")
+    private LogoutTokenStorageService tokenStorageService;
+
+    @Value("${jwt.token.expired:300000}")
```

Comment:
you should transfer it into application.properties. 300000 - hard to read. Do you mean 5 min? I think it very small period. Maybe it only for demo

Reply:
It is just default value in case Spring has not found the property in available property sources.
Nonetheless, it is really hard to read that value, so the property is handled now as number of seconds.
Moreover, the default is set to 10 minutes